ner_v2/tests/pattern/phone_number/phone_number_ner_tests.yaml

```diff
@@ -0,0 +1,108 @@
+tests:
+  en:
+  - _id_: en_1
+    message: Set a reminder on 02226129854
+    outputs:
+    - original_text: '02226129854'
+      output_id: '1'
+      value: '02226129854'
+  - _id_: en_2
+    message: Set a reminder on 022 26129854
+    outputs:
+    - original_text: 022 26129854
```

Comment:
Avoid inconsistency, some original_text are quoted, some are not.

Reply:
Okay